client/components/map/Marker/MarkerIconWrapper.tsx

```diff
@@ -0,0 +1,76 @@
+// General component that controls the display of the markers
+// If clustered, "label" is passed in as an argument
+// If not clustered, "label" is not passed in as an argument
+import { LatLngExpression } from "leaflet";
+import { useMemo } from "react";
+import { Box, Text, Image } from "@chakra-ui/react";
+import { IconProps } from "@chakra-ui/icons";
+
+export interface CustomMarkerProps {
+	position: LatLngExpression;
+	icon?: React.ComponentType<IconProps>; // Update the type here
+	color: string;
+	label?: string;
+	category: string;
+}
+
+const MarkerIconWrapper = ({ icon, color, label, category }: Partial<CustomMarkerProps>) => {
+	const IconFC = useMemo(() => icon ?? null, [icon]);

```

Comment:
don't think this is necessary? we can just use icon direct?

if the concern is that icon is not capitalised, maybe we can use and alias

`{icon: IconFC, ...` in the props on line 17